src/ucp/core/ucp_worker.c

```diff
@@ -679,6 +679,9 @@ ucp_worker_iface_error_handler(void *arg, uct_ep_h uct_ep, ucs_status_t status)
     if (ucp_worker_is_uct_ep_discarding(worker, uct_ep)) {
         ucs_debug("UCT EP %p is being discarded on UCP Worker %p",
                   uct_ep, worker);
+        /* flush cancel operation might be on pending queue due to
+         * UCS_ERR_NO_RESOURCES, so need to purge the queue to resubmit the
+         * operation */
```

Comment:
i think it's "abort the operation", not resubmit, right?

Reply:
`ucp_request_send_state_ff`, in case of discard, has to re-submit the operation to avoid reordering, flush cancel must be completed last otherwise we can get error WQE when lanes are destroyed

Reply:
Add to comment:
/* We need to resubmit the FLUSH_CANCEL operation on the same failed lane, in order to make sure all previous outstanding operations are completed before destroying the failed endpoint */